### ISSUE TITLE
Add search to filter fields in TSA

### ISSUE THREAD

**Issue by SRGDamia1** [ODM2/ODM2DataSharingPortal#382]
![image](https://user-images.githubusercontent.com/5374350/56745109-c65f8b00-6747-11e9-9e02-63f0fea259e4.png)

**Comment by aufdenkampe** [comment]
@SRGDamia1, we'll need to collectively decide how to manage work on the WebTSA application, which is being developed in the `master-envirodiy` branch of the repo at https://github.com/UCHIC/WEBTSA/tree/master-envirodiy.

We at LimnoTech aren't co-developers of that WEBTSA repo, so we'll either need to fork that branch or discuss with @horsburgh how to co-develop in that repo.

Either way, this new feature won't be high on the list of pressing priorities.

**Comment by SRGDamia1** [comment]
I would very much prefer all the broken things get fixed first.  And, as far as that goes, that there be a gui for csv upload.  But, this would be a nice little feature to have.

**Comment by aufdenkampe** [comment]
Yes, bug fixes will be the top priority, including making it as easy as possible for CSV uploads.

**Comment by dbressler75** [comment]
As a follow up to @SRGDamia1's initial comment, in addition to search and filter it would be nice to be able to sort the columns (e.g., alphabetically by Site) - probably already implied but just to be clear.

**Comment by aufdenkampe** [comment]
We have decided to replace TSA with a Time Series Visualization (TSV) page, starting with:
- #512

Check out the screenshots I just posted with https://github.com/ODM2/ODM2DataSharingPortal/issues/512#issuecomment-957935317. 

We have not yet added a search field, but we will enable search by site prior to our first release next month. I'm closing this for now. We can open a new feature request that more clearly defines the need for the new TSV.
